Gemfile

```diff
@@ -4,5 +4,8 @@ group :test do
   gem 'rake'
   gem 'rspec'
   gem 'beaker', '~>1.17.0'
+  if ENV['GEM_SOURCE']
```

Comment:
Seems like kind of an overloaded use of the variable here, then.  Would be nice if we had something more clear, like "if PL_INTERNAL" or something.  GEM_SOURCE could be set by users outside of PL, right?  If we are going to condition based on that var it seems like maybe we should check the value too, and see if it's pointing to our internal gem server.

I don't care too much, won't put up a huge fight, but this doesn't seem like quite the right thing to condition on.